native-emulation/run.sh

```diff
@@ -28,8 +28,8 @@ run_qemu () {
     -device usb-kbd \
     -device 'usb-net,netdev=net0' \
     -netdev 'user,id=net0,hostfwd=tcp::5022-:22' \
-    -sd "$IMAGE_FILE" \
-    -dtb "$PTB_FILE" \
+    -drive "file=$IMAGE_FILE,index=0,format=raw" \
```

Comment:
Avoid warnings that say to use raw mode for the downloaded image.

Reply:
Thank you!

```suggestion
    -drive "file=${IMAGE_FILE},index=0,format=raw" \
```